config/jest/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   testPathIgnorePatterns: ['<rootDir>[/\\\\](dist|node_modules)[/\\\\]'],
   moduleNameMapper: {
-    '\(seek-style-guide\/react|\.(css|less)$)': require.resolve(
+    '(seek-style-guide/react|.(css|less)$)': require.resolve(
```

Comment:
From my understanding. And I could be wrong. `\(` is the same as `(`, the `\` is escaping the next character but the `(` doesn't need to be escaped. I would expect this to need to be `\\(`, as in a literal `\` then a literal `(`.
I'll do some testing and get back to you.
If correct, i'd probably prefer to change it to `\\` as we'll be fighting prettier otherwise.